src/decoder/symphonia.rs

```diff
@@ -77,16 +77,23 @@ impl SymphoniaDecoder {
         };
 
         // Select the first supported track
-        let track_id = probed.format.tracks()
+        let track_id = probed
+            .format
+            .tracks()
             .iter()
-            .find(|t| t.codec_params.codec != CODEC_TYPE_NULL).unwrap().id;
-
-        let decode_opts = Default::default();
+            .find(|t| t.codec_params.codec != CODEC_TYPE_NULL)
+            .unwrap()
+            .id;
 
-        let track = probed.format.tracks().iter().find(|track| track.id == track_id).unwrap();
+        let track = probed
+            .format
+            .tracks()
+            .iter()
+            .find(|track| track.id == track_id)
+            .unwrap();
 
-        let mut decoder = symphonia::default::get_codecs()
-            .make(&track.codec_params, &decode_opts)?;
+        let mut decoder =
+            symphonia::default::get_codecs().make(&track.codec_params, &Default::default())?;
```

Comment:
another nitpick (feel free to ignore if you disagree). I personally find: `Type::trait_fn` clearer then `Trait::trait_fn()`. I think the former would be `DecoderOpt::default()` here.